test/addons/rbd-mirror/test

```diff
@@ -72,11 +68,7 @@ def test_volume_replication(primary, secondary):
     )
 
     print(f"Deploying vr vr-sample in cluster '{primary}'")
-    kubectl.apply(
-        "--filename=vr-sample.yaml",
-        "--namespace=rook-ceph",
-        context=primary,
-    )
+    kubectl.apply("--kustomize=test-data", context=primary)
```

Comment:
Not needed since you already applied test-data before.

Reply:
Done